src/types/line.js

```diff
@@ -273,17 +273,9 @@ LineAnnotation.defaults = {
     yAdjust: 0,
     yPadding: undefined, // TODO: v2 remove support for yPadding
   },
-  scaleID: undefined,
   shadowBlur: 0,
   shadowOffsetX: 0,
   shadowOffsetY: 0,
-  value: undefined,
```

Comment:
I think `endValue` is still in the line defaults. Should be removed as did for `value`.